github-orgs/kubeflow/org.yaml

```diff
@@ -973,6 +973,22 @@ orgs:
             - Bobgy
             - paveldournov
             - theadactyl
+          release-docs-leads:
```

Comment:
Website approver should be able to help with creating 1.5 branch for release team. I don't think we need a new team for the release specifically.

Reply:
@zijianjoy I don't have a strong opinion. Since we give extra permissions to the release manager, this means the release team should have sufficient permission. The release manager can coordinate with the docs leads for cutting the tags.

I think it's more important to move on with this PR and https://github.com/kubeflow/internal-acls/pull/511, in order to be able to tag all members of the release team in the following issues.

So I also agree with 
1. dropping the `release-docs-leads`
2. just introducing a `release-managers` group
3. having a group for the entire release team with no extra permissions https://github.com/kubeflow/internal-acls/pull/511

Reply:
Sounds good to me! I will look at #511 after this PR is merged.

Reply:
Removed `release-docs-leads` - PTAL, thank you!